packages/apollo-components/src/DataTable/useDataTable.tsx

```diff
@@ -54,9 +45,17 @@ export function useDataTable<T>(props: UseTableData<T>): Table<T> {
       columnVisibility,
     },
     defaultColumn: {
-      cell: (cell) => <TypographyCustom noWrap>{cell.getValue() as any}</TypographyCustom>,
+      cell: ({ cell }) => {
+        let truncateMode = cellConfig?.truncateMode ?? 'hover'
+        if (typeof truncateMode === 'function') truncateMode = truncateMode(cell)
```

Comment:
Hmm, I would have written it like this:
```suggestion
      cell: ({ cell }) => {
        if (!cellConfig?.truncateMode) return 'hover'
        const truncateMode = typeof cell.truncateMode === 'function' ? truncateMode(cell) : cell.truncateMode
```

Reply:
Yes 👀